bitcoin/src/lib.rs

```diff
@@ -137,7 +137,7 @@ pub use crate::consensus::encode::VarInt;
 pub use crate::crypto::key::{self, PrivateKey, PublicKey};
 pub use crate::crypto::{ecdsa, schnorr};
 pub use crate::error::Error;
-pub use crate::hash_types::*;
+pub use crate::hash_types::{Txid, Wtxid, BlockHash, Sighash, PubkeyHash,  ScriptHash, WPubkeyHash, WScriptHash};
```

Comment:
I'd like to also remove `Sighash` since it usually doesn't need to be named even when used and we want to avoid potential name conflicts.

Reply:
Ah of course, you'd think after all the re-naming PRs I've done on sighash this past week I would have noticed that. Thanks

Reply:
Sure thing.

Reply:
Oh, in #1597 I never touched `Sighash` - it should be moved over to the `crypto` module. I'll remove the re-export here and fix it over there.